git-ssh.md

```diff
@@ -0,0 +1,75 @@
+# Installing SSH Key on Macbook for GitHub access
+
+I'm not going to walk through the reasoning behind using SSH keys as authorization or why GitHub is making this change. Let's just dive into the process and get the SSH keys created, installed, and referenced on GitHub.
+
+## Create the SSH Key
+1. Open terminal. **I recommend using the terminal application on your MacBook rather than the integrated terminal in VS Code.**
+1. Paste the following code into the terminal, substituting in your GitHub account email address for the `[your GitHub email]`:
+  ```
+  $ ssh-keygen -t -rsa -b 4096 -C "[your GitHub email]"
```

Comment:
This command doesn't work for me. I get 

> unknown key type -rsa

Let's follow the instructions from Github: https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent

> ssh-keygen -t ed25519 -C "your_email@example.com"